### ISSUE TITLE
Cant make leechers detect leader if i run autobaal first

### ISSUE THREAD

**Issue by oband2** [kolton/d2bot-with-kolbot#1628]
Hey!

I've tried edit my script into running baal first, and then diablo.
But the wakka will not work that way. it just says leader not found.
Autodetect works for autobaal, but not for wakka, in this order. If i run diablo first, it works...

What to fix? It says error in line 50 in wakka.js, leader not found..
How can i fix this?

**Comment by oband2** [comment]
fixed

**Comment by oband2** [comment]
https://gyazo.com/c7b477d50ec57e2dcb4594f0cf632e7a

Picture of console, it finds leader for autobaal, but not for wakka, what should i look into?

**Comment by Jcarey89** [comment]
Config.Leader = ""; // Leader's ingame character name. Leave blank to try auto-detection (works in AutoBaal, Wakka, MFHelper)

did u put ur leaders name there?
(in ur char config)

**Comment by oband2** [comment]
> ```
> Config.Leader = ""; // Leader's ingame character name. Leave blank to try auto-detection (works in AutoBaal, Wakka, MFHelper)
> ```
> 
> did u put ur leaders name there?
> (in ur char config)

Yes ofc, as u seen in uploaded picture it did autobaal without problems with the leader.
